example_buffer.py

```diff
@@ -0,0 +1,198 @@
+import tensorflow as tf
+import os
+import random
+from tqdm import tqdm
+import multiprocessing as mp
+import functools
+import itertools
+import rl_loop
+import dual_net
+import subprocess
+from utils import timer
+import time
+import preprocessing
+import argh
+from collections import deque
+import argparse
+import datetime as dt
+
+
+# 1 for ZLIB!  Make this match preprocessing.
+#TODO: Unify with preprocessing
+READ_OPTS = tf.python_io.TFRecordOptions(1)
+
+LOCAL_DIR = "data/"
+
+
+def pick_examples_from_tfrecord(filename, samples_per_game=4):
+    protos = list(tf.python_io.tf_record_iterator(filename, READ_OPTS))
+    if len(protos) < 20:  # Filter games with less than 20 moves
```

Comment:
The 'pass ban' will be somewhat different for e.g. 9x9s.  Plus, files get truncated, GCS randomly dies, whatever... it'd be nice to confirm a minimum amount of length.

Also added min (4, game_length // 50).  This may have a side effect of overweighting recent games.  There are plenty of games that are over after 80-199 moves